app/BotMan/ConfigParser.php

```diff
@@ -37,10 +45,59 @@ public function configure(BotMan $bot)
      *
      * @param string $reply
      */
-    private function reply(string $reply)
+    private function addReply(string $reply) : Callable
     {
         return function (BotMan $bot) use ($reply) {
             $bot->reply($reply);
         };
     }
+
+    /**
+     * Add the Conversation String to the bot.
+     *
+     * Note: The Conversation string must include the fully qualified class
+     * name followed by the `@` symbol and the class method.
+     *
+     *     App\Conversation\MyConversation@handle
+     *
+     * @param string $conversation The conversation string.
+     * @return string $conversation The conversation string.
+     */
+    private function addConversation(string $conversation) : string
+    {
+        if (!$this->isConversationValid($conversation)) {
+            throw new \UnexpectedValueException(sprintf(
```

Comment:
Is there a reason you didn't import `UnexpectedValueException`?

Reply:
I typically don't import classes in the root namespace.  I personally don't see a point, but I'm not firm on this point.